mix.exs

```diff
@@ -22,7 +22,7 @@ defmodule ExqAtomizeJobArguments.MixProject do
   defp deps do
     [
       {:atomic_map, "~> 0.8"},
-      {:exq, "~> 0.13.0"},
+      {:exq, "~> 0.14.0"},
```

Comment:
This version requirement can probably be made a bit looser, I don't anticipate the middleware API changing drastically.